Task7.py

```diff
@@ -0,0 +1,35 @@
+""" 7 Шоколадка **
+Шоколадка имеет вид прямоугольника, разделенного на n×m долек.
+ Шоколадку можно один раз разломить по прямой на две части.
+Определите, можно ли таким образом отломить от шоколадки ровно k долек.
+Определите, можно ли отломить от шоколадки ровно k одиночных долек
+Определите, можно ли отломить от шоколадки ровно k одиночных долек
+с помощью m разломов"""
+
+n = int(input())
+m = int(input())
+k = int(input())
+
+if (n % 2 == 0 or m % 2 == 0) and (n * m) % k == 0:
```

Comment:
if will take 10*12 we can't cut it for 15 but you program prints we can

Reply:
I reasoned this way: if 120 is divisible by 15 without a remainder (120/15 = 8) then we can. if will take 10 * 12 we can cut it for 15. I still have to think about this task.